### ISSUE TITLE
.prompt method from Inquirer module is not working correctly

### ISSUE THREAD

**Issue by yurimalheiros** [accraze/make-license#68]
I tried the application on Mac OS and it does not generate any LICENSE file.
I investigated and I discovered that the .prompt method is called incorrectly. The method returned a promise, so, based on the documentation of the Inquirer module, it should be used this way:

```

inquirer.prompt([/* Pass your questions in here */]).then(function (answers) {
    // Use user feedback for... whatever!! 
});
```

**Comment by yurimalheiros** [comment]
I am fixing this

**Comment by accraze** [comment]
thanks @yurimalheiros I just checked as well and it looks like there were some changes to inquirer in the last update.

**Comment by limenet** [comment]
@yurimalheiros and @accraze see https://github.com/accraze/make-license/pull/65#issuecomment-250933688

**Comment by accraze** [comment]
Issue resolved by #69
